ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/PartyManagementServiceIT.scala

```diff
@@ -90,6 +92,23 @@ final class PartyManagementServiceIT extends LedgerTestSuite {
     }
   })
 
+  test(
```

Comment:
Maybe also test that party identifiers containing invalid characters are rejected (with the correct description)?

Reply:
Absolutely, I'll add one, thanks for pointing this out.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8642/commits/8abe8ee690a8d192fb9634430242e9cb2f36a0e2

@remyhaemmerle-da Is there something else I should use as a test case to ensure we're future proof?

Reply:
I think it is fine like that.